arelle/plugin/validate/EDINET/PluginValidationDataExtension.py

```diff
@@ -175,6 +189,13 @@ def isCorporateForm(self, modelXbrl: ModelXbrl) -> bool:
     def isCorporateReport(self, modelXbrl: ModelXbrl) -> bool:
         return self.jpcrpNamespace in modelXbrl.namespaceDocs
 
+    def isExtensionUri(self, uri: str, modelXbrl: ModelXbrl) -> bool:
+        if uri.startswith(modelXbrl.uriDir):
+            return True
+        if not any(uri.startswith(taxonomyUri) for taxonomyUri in STANDARD_TAXONOMY_URLS):
+            return True
+        return False
```

Comment:
nit: return the result of any directly instead of using an if statement.
```suggestion
        return any(uri.startswith(taxonomyUri) for taxonomyUri in STANDARD_TAXONOMY_URLS)
```